src/third_party/wiredtiger/src/include/misc.h

```diff
@@ -87,6 +87,11 @@
  */
 #define	WT_HASH_ARRAY_SIZE	512
```

Comment:
Is this still needed ? If not, it might be good to comment out just to see if we missed any places.

Reply:
Yes, this one is still used for the size of the encryptor hash array, which should stay at 512.